doc/source/user_guide/index.rst

```diff
@@ -222,4 +222,4 @@ with it. For example:
         recommended.
 
 
-For stability considerations, see :ref:`ref_pymapdl_stability`.
+For stability considerations, see :ref:`PyMAPDL stability <ref_pymapdl_stability>`.
```

Comment:
This format is not correct I think.

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

Reply:
The format used is the following one: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-custom-link-text